control/exception.py

```diff
@@ -39,32 +39,44 @@
 #
 # $Id$
 
+
 class ControlSlycot(Exception):
     """Exception for Slycot import.  Used when we can't import a function
     from the slycot package"""
-    pass
+    def __init__(self, error):
```

Comment:
I don't think it is the right approach to immediately print this error reason. This creates an exception with a by-product.

This also does not call Exception.__init__, (which would normally store the error string) so if caught and printed later, the original exception reason has disappeared.